### DISCUSSION TITLE
AE9F855A-EF81-2BD5-B9AC-6B9E7BE29749

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#196] [General]
# AE9F855A-EF81-2BD5-B9AC-6B9E7BE29749

This add-on adds a realistic, fully 3D model of the planetary nebula Abell 14. Abell 14 is a bipolar nebula somewhat resembling two donuts stacked on top of each other. Overall, it is highly symmetrical and reddish in color.

https://celestia.mobi/addon?item=AE9F855A-EF81-2BD5-B9AC-6B9E7BE29749&lang=pt

**Comment by Alaxkibz** [comment]
legal
